multiselect.go

```diff
@@ -27,6 +27,9 @@ type MultiSelect struct {
 	Default       []string
 	Help          string
 	PageSize      int
+	VimMode       bool
```

Comment:
This field isn't needed anymore, right?

Reply:
Yeah - the VimMode keeps track of if you're in vim mode or not (so its toggled by the `Esc` key). It also lets users start in Vim mode if they wanted

Reply:
Hmm allowing the developer to specify starting in vim-mode is an interesting idea. I wonder if that is something a developer should be able to decide for their users or if we should just default to being not in vim-mode.

Reply:
I don't have a strong feeling either way TBH - only for my use cases I want to be in 'filter' mode OOTB :) But am sure a vim fan might disagree!

Reply:
I think having vim mode be off by default is fine and we can make `VimMode` a private field. As part of another PR, i'd like to improve the messaging in the select and MultiSelect prompts to do 2 things:
1. Indicate to the user when there are options above and below the current cursor
1. Notify them of which mode they're in and general instructions for how to navigate.

I'm curious what @tsuyoshiwada thinks about this discussion.